test/stacktraces.jl

```diff
@@ -168,3 +168,19 @@ end
 @test bt[2].line == 42
 @test bt[2].file === :foo
 end
+
+@noinline f33065(x; b=1.0, a="") = error()
+let bt
+    try
+        f33065(0.0f0)
+    catch
+        bt = stacktrace(catch_backtrace())
+    end
+    @test any(s->startswith(string(s), "f33065(::Float32; b::Float64, a::String)"), bt)
+    try
+        f33065(0.0f0, b=:x)
+    catch
+        bt = stacktrace(catch_backtrace())
+    end
+    @test any(s->startswith(string(s), "f33065(::Float32; b::Symbol, a::String)"), bt)
+end
```

Comment:
should add some tests for a vararg kwfunc too (e.g. `f33065(x; c..., b=1.0, a="")`)

Reply:
I'll be darned, that triggers a lowering error. I guess everybody puts the keyword vararg last :joy: Maybe an unintentional feature?